packages/blade/src/components/Checkbox/CheckboxLabel.web.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
what is diff between `checkboxlabel` and `checkboxlabeltext`?

Reply:
CheckboxLabel renders native `label` which wraps the checkbox.

CheckboxLabelText just renders the text content.